tools/configure.c

```diff
@@ -1590,6 +1596,37 @@ static void refresh(void)
     }
 }
 
+static void save_original_config(void)
+{
+  snprintf(g_buffer, BUFFER_SIZE, "%s%c.config", g_topdir, g_delim);
+  char *source_config = strdup(g_buffer);
+  snprintf(g_buffer, BUFFER_SIZE, "%s%c.config.orig", g_topdir, g_delim);
```

Comment:
why need save config.orig file

Reply:
Because I've tested it and the .config.old file is used differently by kconfig-frontends and kconfiglib. We cannot assume how it works.